lib/quic/frame/frame_types.ex

```diff
@@ -0,0 +1,108 @@
+defmodule QUIC.Frame.Type do
+  @moduledoc """
+  All QUIC Frame types.
+
+  [Source](https://quicwg.github.io/base-drafts/draft-ietf-quic-transport.html#frames)
+  """
+
+  use Bitwise
+
+  @spec padding() :: 0
```

Comment:
I think these might be easier to read as hex values, it looks like the docs prefer defining values in hex as well.

Reply:
Fixed.